CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 
 Breaking changes
 
+- cmd/bnsd: `nft/username` allows now for any number of aliases/names for a
```

Comment:
Please add that there is no longer a reverse lookup from (chainid, address) to name as this would be meaningless without a uniqueness constraint